.github/workflows/push_main_production.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Push CI Production
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  analyze-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: 'stable'
+      - run: flutter pub get
+      - run: flutter pub run build_runner build --delete-conflicting-outputs
+      - run: flutter build apk --release -t lib/main_production.dart
+      - run: flutter analyze
```

Comment:
line 27 & 28 need to be swapped, analyze should run before the build?

Reply:
`pull_request_validation.yml` needs to be changed too

Reply:
@utpal-barman I agree with your suggestion. But Can you please explain why it is kept like this? 
https://github.com/monstar-lab-bd/flutter-template/blob/master/.github/workflows/pull_request_validation.yml

Reply:
known issue to me, I always had to change these lines